SocialFilter/sections/whitelist.txt

```diff
@@ -12,6 +12,11 @@ designervn.net#@#.fb-share-button
 !#endif
 ! #End Social locker
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79915
+gsm1x.xyz#@#.social-icons
+gsm1x.xyz#@##fb-root:not(body):not(html)
+gsm1x.xyz#@#.fb-page
+@@||facebook.com/v4.0/plugins/page.php$domain=gsm1x.xyz
```

Comment:
There is no need to unblock the social widget. Remove leftover.

Reply:
How is that a social widget? Convince me.

Reply:
It's a Facebook Fanpage.

Reply:
> How is that a social widget?

I don't undestand why it's not a social widget? We have a policy to block such elements. In addition, it contains the `Like` and `Share` buttons.

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/114179978-4fc5f300-9948-11eb-840b-a8d8075aa351.png)

</details><br/>

Reply:
Convinced enough. Thank you.

Reply:
You are welcome.